File: src/main/java/com/partymakers/shareparty/domain/party/entity/Expense.java
package com.partymakers.shareparty.domain.party.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Data
@AllArgsConstructor
@EqualsAndHashCode(of= {"name", "cost", "count"})
public class Expense {
    final String  name;
    final Integer cost;
    final Double  count;
}

File: src/main/kotlin/com/partymakers/shareparty/party/presentation/dto/Expense.kt
package com.partymakers.shareparty.party.presentation.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class Expense(
    @JsonProperty("name")
    val name: String,
    @JsonProperty("cost")
    val cost: Int,
    @JsonProperty("count")
    val count: Double
) 